index.html

```diff
@@ -0,0 +1,278 @@
+              <img
+                src="./assets/images/titleicon.png"
+                class="image-logo"
+                alt="Erriecha Logo"
+              />
+            </a>
+          </li>
+          <li class="list-logo list-close" id="popup-closeid">
+            <a href="#" class="link-logo close-icon">&#215;</a>
+          </li>
+        </ul>
+        <ul class="order-primary">
+          <li class="list-primary">
+            <a href="./about.html" class="link-primary link-about">About</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Program</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Join</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Sponsor</a>
+          </li>
+          <li class="list-primary list-news">
+            <a href="" class="link-primary">News</a>
+          </li>
+          <li class="list-primary list-orgname">
+            <a href="" class="link-primary link-orgname">Erriecha </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="align-section section-primary">
+        <div class="humberger-container mobile-only">
+          <a href="javascrip:void()" class="humbergur-link" id="humbergurid">
+            &#9776;</a
+          >
+        </div>
+        <div class="card-primary">
+          <h1 class="event-motto">
+            <q class="motto-quot">Irreechaa Love,Peace and Unity</q>
+          </h1>
+
+          <h1 class="event-title">CELEBRATING IRREECHAA <br />FESTIVAL 2023</h1>
+          <p class="event-message">
+            A joyful celebration believing in value of openness and sharing.
+            Creating a positive change with people from every corner of the
+            countries is taking place in October in Ethiopian capital Finfinnee.
+          </p>
+          <div class="head-venue">
+            <h3 class="venue-date">2023,10,15(sat)-16(sun)</h3>
+            <p class="venue-location">
+              @Finfinnee, Hora Finfinnee and Bishooftuu, Hora Arsadii
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="align-section program-section">
+        <h1 class="program-head">Main Program</h1>
+        <div class="program-container">
+          <ul class="program-card">
+            <li class="program-list program-card-icon">
+              <img
+                src="./assets/images/program/assembly.png"
+                class="program-title-icon"
+              />
+            </li>
+            <li class="program-list program-card-title">
+              <h3>Lecturer</h3>
+            </li>
+            <li class="program-list program-card-content">
+              <p>
+                Listen to the speakers from various countries about the message
+                of sharing and opening.
+              </p>
+            </li>
+          </ul>
```

Comment:
- [ ] Kindly use the `article` tag for this section as it is a self-contained composition on the page. The semantic `<article>` tag in HTML is used to mark up self-contained content that can be independently distributed or reusable, such as news articles, blog posts, user-submitted comments, forum posts, and product reviews. It helps search engines and screen readers to identify and distinguish main content from other page elements, such as navigation menus, sidebars, and footers. 🤝️
- [ ] This goes for all 5 cards.
You can check this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to get more insight on when to use the semantic article element.

Reply:
- [ ] @osmanwako great work implementing the changes requested however, you forgot to add the `<article>` tag in each of the 5 cards, kindly add it.